fjord/feedback/jinja2/feedback/android_feedback.html

```diff
@@ -0,0 +1,192 @@
+{% extends "base.html" %}
+
+{% block extra_headers %}
+  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=no" />
+{% endblock %}
+
+{% block site_js %}
+  {% javascript 'android_feedback' %}
+{% endblock %}
+
+{% block site_css %}
+  {% stylesheet 'android-feedback' %}
+{% endblock %}
+
+{% block page_title %}{{ _('Firefox Feedback') }}{% endblock %}
+
+{% set extra_body_attrs = {'data-form-name': 'android'} %}
+
+{% block body_id %}android{% endblock %}
+
+{% block body %}
+  <div class="main">
+    {% block content %}
+        <div class="content">
+
+          <div class="deck">
+            <div class="card no-back" id="intro">
+              <section>
+
+                <h1>
+                  {% trans %}
+                    Have a minute?
+                  {% endtrans %}
+                </h1>
+                <div class="message">
+                  {% trans product=product.display_name %}
+                    Tell us what you think about {{product}} so far.
+                  {% endtrans %}
+                </div>
+
+                <noscript>
+                  <p class="error">
+                    {% trans %}
+                      JavaScript is required to leave feedback. Please enable JavaScript in your browser and refresh this page.
+                    {% endtrans %}
+                  </p>
+                </noscript>
+
+                <div id="happy-button" class="link-box sentiment-button" tabindex=0>
+                  <a>
+                    {% trans %}
+                      I love it
+                    {% endtrans %}
+                  </a>
+                </div><!-- #happy-button -->
```

Comment:
Instead of making these HTML comments which will show up in the output, you could make them Jinja2 comments which won't.